<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VivaPassLock</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(23, 33, 43);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1101</width>
      <height>621</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>471</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(31, 41, 54);
border-radius: 30px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>171</width>
         <height>151</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="base.qrc">:/new/icon/Resources/assets/icon/password.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>200</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 24pt &quot;Nirmala UI&quot;;
color: &quot;#30C193&quot;;</string>
       </property>
       <property name="text">
        <string>PassLock</string>
       </property>
      </widget>
      <widget class="QPushButton" name="createButton">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>350</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 13pt &quot;Nirmala UI&quot;;
color: &quot;#30C193&quot;;
border: none;</string>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>200</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 24pt &quot;Nirmala UI&quot;;
color: rgb(141, 151, 173);</string>
       </property>
       <property name="text">
        <string>Viva</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loginButton">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>280</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#loginButton {
border: 2px solid &quot;#30C193&quot;;
border-radius: 25px;
color: &quot;#30C193&quot;;
font: 700 12pt &quot;Nirmala UI&quot;;
}
#loginButton:hover {
background-color: &quot;#30C193&quot;;
color: rgb(31, 41, 54);
}</string>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5"/>
    <widget class="QWidget" name="page_4">
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>471</width>
        <height>441</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(31, 41, 54);
border-radius: 30px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>200</width>
         <height>200</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="GoButton">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>370</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#GoButton {
border: 2px solid &quot;#30C193&quot;;
border-radius: 25px;
color: &quot;#30C193&quot;;
font: 700 12pt &quot;Nirmala UI&quot;;
}</string>
       </property>
       <property name="text">
        <string>Go to Login  </string>
       </property>
       <property name="icon">
        <iconset resource="base.qrc">
         <normaloff>:/new/icon/Resources/assets/icon/arrow-right.png</normaloff>:/new/icon/Resources/assets/icon/arrow-right.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>200</width>
         <height>200</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>260</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 24pt &quot;Nirmala UI&quot;;
color: &quot;#30C193&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>471</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(31, 41, 54);
border-radius: 30px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>200</width>
         <height>200</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="createPassword">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>220</y>
         <width>351</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid &quot;#30C193&quot;;
border-radius: 14px;
font: 18pt &quot;Nirmala UI&quot;;
color: &quot;#30C193&quot;;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>21</number>
       </property>
       <property name="placeholderText">
        <string> Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="createrePassword">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>290</y>
         <width>351</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid &quot;#30C193&quot;;
border-radius: 14px;
font: 18pt &quot;Nirmala UI&quot;;
color: &quot;#30C193&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>21</number>
       </property>
       <property name="placeholderText">
        <string> Replace password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="creatingButton">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>360</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#creatingButton {
border: 2px solid &quot;#30C193&quot;;
border-radius: 25px;
color: &quot;#30C193&quot;;
font: 700 12pt &quot;Nirmala UI&quot;;
}
#creatingButton:hover {
background-color: &quot;#30C193&quot;;
color: rgb(31, 41, 54);
}</string>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
      <widget class="QPushButton" name="hideandseeButton">
       <property name="geometry">
        <rect>
         <x>359</x>
         <y>230</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="base.qrc">
         <normaloff>:/new/icon/Resources/assets/icon/eye.png</normaloff>:/new/icon/Resources/assets/icon/eye.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="backtohomeButton2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="base.qrc">
        <normaloff>:/new/icon/Resources/assets/icon/back-arrow.png</normaloff>:/new/icon/Resources/assets/icon/back-arrow.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>520</y>
        <width>421</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 61, 87);
font: 700 10pt &quot;Nirmala UI&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>471</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(31, 41, 54);
border-radius: 30px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="idEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>220</y>
         <width>351</width>
         <height>51</height>
        </rect>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid &quot;#30C193&quot;;
border-radius: 14px;
font: 18pt &quot;Nirmala UI&quot;;
color: &quot;#30C193&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="placeholderText">
        <string> ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="passEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>290</y>
         <width>351</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid &quot;#30C193&quot;;
border-radius: 14px;
font: 18pt &quot;Nirmala UI&quot;;
color: &quot;#30C193&quot;;</string>
       </property>
       <property name="maxLength">
        <number>19</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string> Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loginingButton">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>360</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#loginingButton {
border: 2px solid &quot;#30C193&quot;;
border-radius: 25px;
color: &quot;#30C193&quot;;
font: 700 12pt &quot;Nirmala UI&quot;;
}
#loginingButton:hover {
background-color: &quot;#30C193&quot;;
color: rgb(31, 41, 54);
}</string>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>200</width>
         <height>200</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="copyidButton">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>230</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#copyidButton {
border: none;
color: &quot;#30C193&quot;;
font: 700 12pt &quot;Nirmala UI&quot;;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="base.qrc">
         <normaloff>:/new/icon/Resources/assets/icon/copy.png</normaloff>:/new/icon/Resources/assets/icon/copy.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>19</width>
         <height>19</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="hideandseeButton_2">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>300</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="base.qrc">
         <normaloff>:/new/icon/Resources/assets/icon/hidden.png</normaloff>:/new/icon/Resources/assets/icon/hidden.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="backtohomeButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="base.qrc">
        <normaloff>:/new/icon/Resources/assets/icon/back-arrow.png</normaloff>:/new/icon/Resources/assets/icon/back-arrow.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(31, 41, 54);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="minimizeButton">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>0</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#minimizeButton {
border: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="base.qrc">
       <normaloff>:/new/icon/Resources/assets/icon/min.png</normaloff>:/new/icon/Resources/assets/icon/min.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="noneButton">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>0</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="base.qrc">
       <normaloff>:/new/icon/Resources/assets/icon/max.png</normaloff>:/new/icon/Resources/assets/icon/max.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>0</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#closeButton {
border: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="base.qrc">
       <normaloff>:/new/icon/Resources/assets/icon/close.png</normaloff>:/new/icon/Resources/assets/icon/close.png</iconset>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="base.qrc"/>
 </resources>
 <connections/>
</ui>
